### ISSUE TITLE
Loader upon brush or click

### ISSUE THREAD

**Issue by jgmurillo10** [john-guerra/navio#23]
Change pointer to loading `on brush` and `on click`. Currently is working `on sort`

**Comment by john-guerra** [comment]
Tried but simply couldn't do it.

John.
From my phone, brevity and all
http://johnguerra.co

**Comment by john-guerra** [comment]
Revisited as part of a 2026-08-01 whole-repo review. Traced why brush/click interactions feel slow: brush-drag itself is already deferred correctly (only brush-*end* triggers the expensive filter/redraw pipeline), so the likely cause is unconditional `DEBUG=true` console-logging of the full mouse event on every mousemove (#58) plus a scale object rebuilt on every pointer event in the tooltip/hover path (#62) — both filed as separate fixes. Worth re-testing whether a loading indicator is even still needed once those land. See `docs/ai/2026-08-01-repo-review-and-roadmap.md`, §8.
